tourdedave/the-internet
# Unexpected alert dialog

**Change log**
Unexpected browser alerts have the potential to disable a selenium test suite if not handled appropriately. In my experience I have found any unexpected browser alert will fail all other tests in the suite.

I would like to add a unexpected alert scenario to this library that:
1) Upon clicking the link in the unexpected_alert page:
a) Opens a new tab that loads the home page.
b) Loads an alert message when the parent tab is clicked.

**Other changes**
1. Gemfile
Increased Ruby version to 2.4.1.

**Verification**
1. Open terminal, type ```rackup```.
2. Go to http://127.0.0.1:9292/unexpected_alert.
3. Click the link in the page. A new tab will open.
4. Click the parent tab, a browser alert message will appear.
